spec/index.bs

```diff
@@ -1205,6 +1222,44 @@ The default is used if a value is not explicitly specified in {{GPUDeviceDescrip
     <tr class=row-continuation><td colspan=4>
         The maximum allowed {{GPUVertexBufferLayout/arrayStride}}
         when creating a {{GPURenderPipeline}}.
+
+    <tr><td><dfn>maxVertexOutputComponents</dfn>
+        <td>{{GPUSize32}} <td>Higher <td>64
+    <tr class=row-continuation><td colspan=4>
+        The maximum allowed number of components of output variables for a
+        vertex stage {{GPUShaderModule}} entry-point.
+
+    <tr><td><dfn>maxFragmentInputComponents</dfn>
+        <td>{{GPUSize32}} <td>Higher <td>60
+    <tr class=row-continuation><td colspan=4>
+        The maximum allowed number of components of input variables for a
+        fragment stage {{GPUShaderModule}} entry-point.
+
+    <tr><td><dfn>maxColorAttachments</dfn>
+        <td>{{GPUSize32}} <td>Higher <td>4
+    <tr class=row-continuation><td colspan=4>
+        The maximum size of {{GPURenderPassDescriptor/colorAttachments}} and
+        {{GPUFragmentState/targets}}.
+
+        Issue: Do we need to have a max per-pixel render target size?
+
+    <tr><td><dfn>maxComputeWorkgroupStorageSize</dfn>
+        <td>{{GPUSize32}} <td>Higher <td>16352
+    <tr class=row-continuation><td colspan=4>
+        The maximum number of bytes used for a compute stage
+        {{GPUShaderModule}} entry-point.
+
+    <tr><td><dfn>maxComputeWorkgroupInvocations</dfn>
+        <td>{{GPUSize32}} <td>Higher <td>256
```

Comment:
A workaround may not be needed, unless there are a significant number of devices that support 256 but don't support (256,1,1) and (1,256,1) and (1,1,256).